mod/wikirate/spec/set/right/related_article_spec.rb

```diff
@@ -5,8 +5,8 @@
     @sample_company = get_a_sample_company
     @sample_topic = get_a_sample_topic
     @sample_analysis = get_a_sample_analysis
-    @sample_claim = get_a_sample_claim
-    @related_article_card = @sample_claim.fetch trait: :related_articles
+    @sample_claim = get_a_sample_note
+    @related_article_card = Card.fetch @sample_claim.fetch :trait=>:related_articles
```

Comment:
Line is too long. [84/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.